File: src/main/cub/hackerrank/tutorials/a30DaysOfCode/day26/day26.py
#!/usr/bin/env python3

# Enter your code here. Read input from STDIN. Print output to STDOUT

da, ma, ya = list(map(int, input().split(' ')))
de, me, ye = list(map(int, input().split(' ')))

if ya < ye or (ya == ye and ma < me) or (ya == ye and ma == me and da <= de):
    print(0)
elif ya > ye:
    print('10000')
elif ma > me:
    print(500 * (ma - me))
elif da > de:
    print(15 * (da - de))

File: src/main/cub/hackerrank/tutorials/a30DaysOfCode/day26/day26.pl
#!/usr/bin/env perl

# Enter your code here. Read input from STDIN. Print output to STDOUT

my $line;

$line = <STDIN>;
chomp $line;
my ($da, $ma, $ya) = split(' ', $line);

$line = <STDIN>;
chomp $line;
my ($de, $me, $ye) = split(' ', $line);

if ($ya < $ye || ($ya == $ye && $ma < $me) || ($ya == $ye && $ma == $me && $da <= $de)) {
  print(0);
}
elsif ($ya > $ye) {
  print("10000");
}
elsif ($ma > $me) {
  print(500 * ($ma - $me));
}
elsif ($da > $de) {
  print(15 * ($da - $de));
}

print("\n");
